Jenkinsfile.trigger

```
@@ -70,6 +71,7 @@ node {
 									| index($arch)
 								)
 							)
+							| .payload = (gha_payload | @json)
```

I think I'd rather call this `gha_payload` explicitly, and _only_ generate/set/return it when it's actually necessary (which we can then use further down as the determiner so that this becomes the only block that knows/cares about `BASHBREW_ARCH=gha`):

```suggestion
							| if env.BASHBREW_ARCH == "gha" then
								.gha_payload = (gha_payload | @json)
							else . end
```